### ISSUE TITLE
support for srt files in same remote folder on google drive regardless of srt filename

### ISSUE THREAD

**Issue by ddurdle** [ddurdle/GDrive-for-KODI#58]
testing support for and validate+fix if required, support for srt files in same remote folder on google drive regardless of srt filename.

**Comment by hoangphison** [comment]
+1

**Comment by ddurdle** [comment]
Now available in gdrive-testing 0.7.24 for candidate for 0.8 release.

When the setting (under SRT settings) "Load only SRT / SUB from same folder as video" is enabled, plugin will load all SRT/SUB files found in the same directory as the video file.

**Comment by davilaf** [comment]
hi, thanks for this addon is great, buy I cant get srt subtitles work from tv series, I cannot find Load only SRT / SUB from same folder as video in the settings please help

**Comment by ddurdle** [comment]
Can you confirm you can load subtitles contained in the same Google Drive folders as your video?

**Comment by davilaf** [comment]
Yes I have srt external subtitles of each chapter inside a season folder which contains the subtitles files exactly named as the vídeo files

**Comment by ddurdle** [comment]
You can you paste a sample name of the SRT filename here.

**Comment by davilaf** [comment]
Sure look the attached picture
![screenshot](https://user-images.githubusercontent.com/30912012/29196725-f4bd4762-7dfb-11e7-973a-8dfbb272dffb.jpg)

**Comment by ddurdle** [comment]
investigating the issue reported

**Comment by davilaf** [comment]
Hi any reply for this? hope you can fix it
